packages/design-system/scripts/cmsds-migrate/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "cmsds-migrate",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "Migration helper for CMSDS filesystem changes",
+  "license": "ISC",
+  "scripts": {
+    "migrate": "node src/main.js"
+  }
+}
```

Comment:
Now that the deps are moved out, I don't think there's any reason to keep this file.

Reply:
The type: module is important since i'm using imports, it won't run without it and I don't want to mess with the main package.json

Reply:
Ah, okay, weird that it works even though this isn't at the root of the package 🤷‍♂️. I just tested it myself by yarn installing at the root of the repo with and without this file and running `yarn cmsds-migrate` from the root. It's definitely broken without it 👍